src/components/learner-credit-management/cards/data/utils.ts

```diff
@@ -162,7 +162,7 @@ export const isInviteEmailAddressesInputValueValid = ({
       // Check for duplicates (case-insensitive)
```

Comment:
nit: It shouldn't be called lowerCasedEmails, because it seems like we're manually setting them to lowercase from that list